{
	"schema_version": "1.6.0",
	"id": "RHSA-2021:3081",
	"modified": "2024-09-16T06:11:07Z",
	"published": "2024-09-16T06:11:07Z",
	"related": [
		"CVE-2021-3429"
	],
	"summary": "Red Hat Security Advisory: cloud-init security update",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:N"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::appstream",
				"name": "cloud-init",
				"purl": "pkg:rpm/redhat/cloud-init"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:20.3-10.el8_4.5"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2021:3081.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2021:3081"
		},
		{
			"type": "ARTICLE",
			"url": "https://access.redhat.com/security/updates/classification/#moderate"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1940967"
		},
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/security/data/csaf/v2/advisories/2021/rhsa-2021_3081.json"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2021-3429"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2021-3429"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2021-3429"
		}
	]
}
